dashboard/app/models/script_level.rb

```diff
@@ -439,6 +439,13 @@ def summarize(include_prev_next=true)
     summary
   end
 
+  def summarize_for_edit
+    summary = summarize
+    summary[:levelNumber] = activity_section_position
```

Comment:
levelNumber is used by the ProgressBubble to show the number of the level in the lesson. So I think that is still based on the position in the lesson not the activity section. However we were also tracking `position` for level and that might need to be activity_section_position. Happy to help take a closer look or talk about this tomorrow

Reply:
yes, let's discuss. I am not sure which it should be. if I have 6 levels across 3 progressions, which way should they be numbered on the lesson edit page?
```
1
234
56
```
or 
```
1
123
12
```
now that I ask that, the first way does seem a bit more sensible, since that's how it will look on the script overview page. so I think I'm sold on `levelNumber` being `script_level.position`.

Reply:
I don't know if we need to pass this information from the server. Since we already have position I was just setting this up in the jsx component. I don't know if that is bad practice

Reply:
> However we were also tracking `position` for level and that might need to be activity_section_position.

Agreed, if the client is using a position value to order levels within an activity section, then it probably needs to be using the activity section position and not `script_level.position` which would be the position within the lesson. 

I'm going to propose that I hold off on this PR until we have add/update working for script levels which I'm already working on in the next PR.

Reply:
This is now taken care of in #37253 .